test/preparation/data/4NKK_clean.pqr

```diff
@@ -415,8 +415,8 @@ ATOM    414  HB2 ASN    25      -4.842  -2.723  -4.077  0.0000 0.0000
 ATOM    415  HB3 ASN    25      -3.699  -3.859  -4.345  0.0000 0.0000
 ATOM    416  OD1 ASN    25      -3.995  -4.197  -1.476 -0.5500 1.4000
 ATOM    417  ND2 ASN    25      -5.831  -4.622  -2.695 -0.7800 1.5000
-ATOM    418 HD21 ASN    25      -6.311  -4.493  -3.564  0.3900 1.0000
-ATOM    419 HD22 ASN    25      -6.243  -5.168  -1.969  0.3900 1.0000
+ATOM    418 HD22 ASN    25      -6.311  -4.493  -3.564  0.3900 1.0000
+ATOM    419 HD21 ASN    25      -6.243  -5.168  -1.969  0.3900 1.0000
```

Comment:
Why change these?

Reply:
a different version of PDB2PQR numbered the atoms in a bizarre way

Reply:
lol I see